code/python/networkong/pycrowler/crowler_bingcn/saveImage.py

```diff
@@ -58,7 +58,7 @@ def schedule_run():
         now_str = datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S')
         sched_str = sched_time.strftime('%Y-%m-%d %H:00:00')  # schedule to next sharp hour.
         logging.info('now time:%s; next time:%s'% (now_str,sched_str))
-        s.enter(DYNAMIC_CHECK_INTERVAL,1,saveImg,())
+        s.enter(DYNAMIC_CHECK_INTERVAL,1,1,saveImg())
```

Comment:
This should is 
`s.enter(DYNAMIC_CHECK_INTERVAL,1,saveImg(),())`
but 👇can also be run
`s.enter(DYNAMIC_CHECK_INTERVAL,1,1,saveImg())`